region/maridia/outer/Mt. Everest.json

```diff
@@ -1875,6 +2049,114 @@
         "This could be done with many other sets of item combinations, but it would be very tedious for a 1.5 tile runway extension."
       ]
     },
+    {
+      "link": [4, 4],
+      "name": "Leave Spinning (Space Jump)",
+      "requires": [
+        "Gravity",
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveSpinning": {
+          "remoteRunway": {
+            "length": 12,
+            "openEnd": 2,
+            "steepUpTiles": 5,
+            "steepDownTiles": 5
+          },
+          "maxExtraRunSpeed": "$3.8"
+        }
+      }
+    },
+    {
+      "link": [4, 4],
+      "name": "Leave With Mockball (Space Jump)",
+      "requires": [
+        "Gravity",
+        "SpaceJump"
+      ],
+      "exitCondition": {
+        "leaveWithMockball": {
+          "remoteRunway": {
+            "length": 12,
+            "openEnd": 2,
+            "steepUpTiles": 5,
+            "steepDownTiles": 5
+          },
+          "landingRunway": {
+            "length": 4,
+            "openEnd": 1
+          },
+          "maxExtraRunSpeed": "$3.8"
+        }
+      }
+    },
+    {
+      "link": [4, 4],
+      "name": "Leave With Spring Ball Bounce (Space Jump)",
+      "requires": [
+        "Gravity",
+        "SpaceJump",
+        "canTrickyJump"
+      ],
+      "exitCondition": {
+        "leaveWithSpringBallBounce": {
+          "remoteRunway": {
+            "length": 12,
+            "openEnd": 2,
+            "steepUpTiles": 5,
+            "steepDownTiles": 5
+          },
+          "landingRunway": {
+            "length": 4,
+            "openEnd": 1
+          },
+          "movementType": "uncontrolled",
+          "maxExtraRunSpeed": "$3.8"
+        }
+      }
+    },
+    {
+      "link": [4, 4],
+      "name": "Leave Space Jumping",
+      "requires": [
+        "Gravity",
+        "canTrickyJump"
+      ],
+      "exitCondition": {
+        "leaveSpaceJumping": {
+          "remoteRunway": {
+            "length": 12,
+            "openEnd": 2,
+            "steepUpTiles": 5,
+            "steepDownTiles": 5
+          },
+          "maxExtraRunSpeed": "$3.8"
+        }
+      }
+    },
+    {
+      "link": [4, 4],
+      "name": "Leave With Temporary Blue",
```

Comment:
Refined this to describe multiple possible runways that could be used..

Added a couple more suitless cross-room strats now, coming in shinecharging at 1 or 4 and leaving with temporary blue at 2 or 3. I think for the Gravity strats, the in-room runways (along with the 1->5 cross-room one) already provide good coverage? I guess it depends on how much we care about including alternatives to gravity jumping, in case there's some scenario where that's not an option.

Short-charging using the runway at 11 from left-to-right I think is shorter than what we've been putting in elsewhere; haven't been putting in runways that have effective length of 12 tiles or less; we could add them but it seems like Beyond territory? In this case if you wanted to blue space jump to the right door I think you would use the left-most island instead, since that is longer, a little more than 14 tiles. Is it worth adding to avoid a gravity jump?